### DISCUSSION TITLE
Tetris - 3: GameOver

### DISCUSSION THREAD

**Discussion by DrC0okie** [web-classroom/web-classroom.github.io#57] [Q&A : Pratique / Labos] [answered]
Les scores affichés sur la page HTML gérés par la méthode `updateScores`, de Renderer. Cette méthode est appelée dans le gameLoop dans app.js chez moi (j'aurai pu l'appeler dans render(), mais cette méthore aurait eu trop de responsabilité).

Le problème est que lors d'un gameOver, les scores affichés dans l'alerte ne sont pas identiques aux scores affichés sur la page HTML, car quand le GameOverMessage est reçu, les scores n'on pas encore été mis à jour par updateScores et Je n'ai pas d'idée de comment faire en sorte que ces scores se synchronisent.

Doit-on faire en sorte que lors d'un game over, les scores affichés dans l'alerte soient les m^emes que ceux affichés dans la page HTML?

**Comment by Nortalle** [comment, marked as answer]
Bonjour @DrC0okie,

Merci pour votre patience, n'hésitez pas à taguer @web-classroom/web-vgx-teachers la prochaine fois.

Pour répondre à votre question : oui, nous devons faire en sorte que lors d'un game over, les scores affichés dans l'alerte soient les mêmes que ceux affichés dans la page HTML.

`updateScores` a été pensée pour être appelée dans la méthode `render()`, je prends volontiers vos conseils et remarques pour l'année prochaine si vous imaginez une meilleure manière de faire.
